src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -5,9 +5,10 @@
 import com.suse.saltstack.netapi.config.ClientConfig;
 import static com.suse.saltstack.netapi.config.ClientConfig.*;
 import com.suse.saltstack.netapi.config.ProxySettings;
+import com.suse.saltstack.netapi.datatypes.Job;
```

Comment:
Minor nitpick but style changes / import reordering should be done in a separate commit/PR.